crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -799,10 +940,36 @@ async fn validate_warrant_op(
                         )
                         .into());
                     }
-                    action.clone()
+                    (action.clone(), validation_status)
                 };
                 verify_action_signature(action_sig, &action).await?;
 
+                match validation_status {
+                    ValidationStatus::Valid => {
+                        // TODO block the warrant author
+                        return Err(SysValidationError::ValidationOutcome(
+                            ValidationOutcome::InvalidWarrant(
+                                Box::new(op.warrant().clone()),
+                                "Warranted op is valid".into(),
+                            ),
+                        ));
+                    }
+                    ValidationStatus::Abandoned => {
```

Comment:
Can't really do much with this status... it should be implemented everywhere or removed